.github/ISSUE_TEMPLATE/update_saas_webiste.md

```diff
@@ -0,0 +1,89 @@
+---
+name: 🏥 SaaS Website Request
+about: Submit a request form for a new Saas client website
+title: ""
+labels: enhancement
+assignees: ""
+---
+
+## What will you update?
```

Comment:
```
## Update request type
```

Reply:
@sejune-oh 
 I changed the `what will you update` section name.